code/modules/mob/living/living_say.dm

```diff
@@ -26,7 +26,6 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 
 	// Misc
 	RADIO_KEY_AI_PRIVATE = RADIO_CHANNEL_AI_PRIVATE, // AI Upload channel
-	MODE_KEY_VOCALCORDS = MODE_VOCALCORDS,		// vocal cords, used by Voice of God
```

Comment:
Was this mode/key unused ?

Reply:
clerical error I assume?

Reply:
Same reason why there is no `MODE_KEY_ALIEN` or why `MODE_KEY_BINARY` isn't on the list, it's a saymode key and doesn't rely on `GLOB.department_radio_keys`.